src/main/java/net/sf/jabref/model/DuplicateCheck.java

```diff
@@ -16,25 +18,28 @@
 import net.sf.jabref.model.entry.FieldProperty;
 import net.sf.jabref.model.entry.InternalBibtexFields;
 
+import info.debatty.java.stringsimilarity.Levenshtein;
```

Comment:
Just go ahead. No worries. We migrate to another library if the functionality is the same. If not, we just keep the dependency. Reason for migrating is only https://github.com/koppor/jabref/issues/135